extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -8,8 +8,9 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'wikia.document',
 	'wikia.log',
 	'wikia.window',
+	require.optional('ext.wikia.aRecoveryEngine.instartLogic.recovery'),
 	require.optional('ext.wikia.aRecoveryEngine.pageFair.recovery')
-], function (googleSlots, adSlot, slotRegistry, adBlockDetection, doc, log, win, pageFair) {
+], function (googleSlots, adSlot, slotRegistry, adBlockDetection, doc, log, win, instartLogic, pageFair) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "pageFair". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)